packages/sirius-web/backend/sirius-web-papaya/src/main/java/org/eclipse/sirius/web/papaya/services/library/publishers/ReactorLibraryPublisher.java

```diff
@@ -95,6 +96,9 @@ public void publish(PublishPapayaLibraryCommand command) {
             var reactiveStreams003Library = optionalReactiveStreams003Library.get();
             this.createReactor(command, editingDomain.getResourceSet(), List.of(java003Library, reactiveStreams003Library));
         }
+        if (editingDomain.getAdapterFactory() instanceof ComposedAdapterFactory composedAdapterFactory) {
```

Comment:
Not sure it would make a difference in practice, but shouldnt' we test for `instanceof IDisposable` instead? It's what org.eclipse.emf.edit.provider.ComposedAdapterFactory.dispose() does on its individual components.

Reply:
ok, done